server/src/main/java/org/opensearch/identity/ExtensionTokenProcessor.java

```diff
@@ -44,19 +121,37 @@ public PrincipalIdentifierToken generateToken(Principal principal) {
      *
      * @opensearch.internal
      *
-     * This method contains a placeholder implementation.
+     * This method contains a placeholder simplementation.
      * More concrete implementation will be covered in https://github.com/opensearch-project/OpenSearch/issues/4485
      */
     public Principal extractPrincipal(PrincipalIdentifierToken token) throws IllegalArgumentException {
         // check is token is valid, we don't do anything if it is valid
         // else we re-throw the thrown exception
+
+        String token_name = token.getWriteableName();
```

Comment:
Variable names should be in `lowerCamelCase`

Reply:
Oops, old Python habits diehard.